src/round_constants.rs

```diff
@@ -4,6 +4,11 @@ use curve25519_dalek::scalar::Scalar;
 use lazy_static::*;
 
 lazy_static! {
+  /// `ROUND_CONSTANTS` constists on a static reference
+  /// that points to the pre-loaded 960 Scalar constants.
+  /// 
+  /// This 960 `Scalar` constants are loaded from `ark.bin`
+  /// where all of the `Scalar` are represented in bytes.
```

Comment:
Maybe we should mention that the required number of constants is defined by `Width * (RF + Rp)`?

Reply:
In this case, we are far above the minimum for our 603 required constants (width: 9, RF: 8, Rp: 59)

Reply:
We are loading 960 constants but with our params it's true that we require less.

But being strict with which the function does, I don't think it's an error to say that we load 960 in total even if we use less later.